### ISSUE TITLE
Launching minecraft

### ISSUE THREAD

**Issue by kbroman** [jennybc/operation-chromebook#3]
Super helpful notes, thanks!

Executing `Minecraft.jar` directly doesn't work for me. 

I'm finding I need to run `java -jar Minecraft.jar`. I put that in a script preceded by `#/bin/bash`, and `chmod +x` that. Did I miss a step?

Also:
- Does the 16 GB SSD seem enough for you?
- Are the kids satisfied with these chromebooks for minecraft? I think mine are spoiled by fancier hardware.

**Comment by jennybc** [comment]
Welcome @kbroman!

I haven't had a problem with space, but my use of my chromebook is very sporadic and light. It has been useful for seeing ubuntu stuff with my own two eyes (vs indirectly on travis), but I know you have other solutions for that.

The kids are not into Minecraft right now, but they go in and out of periods of heavy use. They do complain about it not being as nice as on the MacBook Pro (no kidding!) but they can tolerate it. They fact that all 3 can play at same time is sometimes worth it to them. The impetus for this was an entire week of Minecraft camp and the chromebooks got the job done.

**Comment by kbroman** [comment]
Makes sense; thanks, @jennybc!

I think I figured out what you're doing, regarding launching minecraft. I noticed in your [photo](https://raw.githubusercontent.com/jennybc/operation-chromebook/master/img/minechrome_04.jpg) that there's a file browser behind the minecraft window. And indeed if I double-click on `Minecraft.jar` in the file browser (provided I made it executable) it does launch. (I had been trying to execute it from the command line.)
